dev-vcs/gitqlient/gitqlient-1.2.0.recipe

```diff
@@ -0,0 +1,110 @@
+SUMMARY="Multi-platform Git client written with Qt"
+DESCRIPTION="GitQlient, pronounced as git+client (/gɪtˈklaɪənt/) is a \
+multi-platform Git client originally forked from QGit. \
+Nowadays it goes beyond of just a fork and adds a lot of new functionality. \
+Some of the major feature you can find are: \
+New features: \
+Easy access to remote actions like: push, pull, submodules management and \
+branches \
+Branches management \
+Tags and stashes management \
+Submodules handling \
+Allow to open several repositories in the same window \
+Better visualization of the commits and the work in progress \
+Better visualization of the repository view \
+Improved UI experience \
+Easy access to the main Git actions \
+Better code separation between Views and Models \
+Simplification of the different options one can do, keeping it to what a Git \
+client is."
```

Comment:
Suggestion (looks better in HaikuDepot) :)
```
DESCRIPTION="GitQlient, pronounced as git+client (/gɪtˈklaɪənt/) is a \
multi-platform Git client originally forked from QGit.
Nowadays it goes beyond of just a fork and adds a lot of new functionality.
Some of the major feature you can find are:
New features:
* Easy access to remote actions like: push, pull, submodules management and \
branches
* Branches management
* Tags and stashes management
* Submodules handling
* Allow to open several repositories in the same window
* Better visualization of the commits and the work in progress
* Better visualization of the repository view
* Improved UI experience
* Easy access to the main Git actions
* Better code separation between Views and Models
* Simplification of the different options one can do, keeping it to what a Git \
client is."
```

Reply:
Done.